auth-server/src/services/auth.ts

```diff
@@ -100,14 +100,9 @@ export default class AuthService {
 
 	private async sendVerificationTokenMail (user: User): Promise<void> {
```

Comment:
Suggest changing the name to `createAndSendEmailVerificationToken`

Reply:
done